dask/highlevelgraph.py

```diff
@@ -148,14 +149,14 @@ def cull(
 
         return MaterializedLayer(out, annotations=self.annotations), ret_deps
 
-    def get_dependencies(self, key: Hashable, all_hlg_keys: Iterable) -> set:
+    def get_dependencies(self, key: Key, all_hlg_keys: Collection[Key]) -> set:
         """Get dependencies of `key` in the layer
 
         Parameters
         ----------
-        key: Hashable
+        key:
```

Comment:
Did you remove the type here on purpose?

Reply:
yes, there was an informal agreement that it is redundant with the type annotations. Happy to ridiscuss.

Reply:
We sometimes use more user-friendly annotations here, but I don't feel strongly about this at all